### DISCUSSION TITLE
Overflowing containers on mobile screens

### DISCUSSION THREAD

**Discussion by RangHo** [community/community#55800] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In Firefox Android on Samsung Galaxy S23 Ultra, the individual containers for repositories are way too large.

![Screenshot_20230518_105459_Firefox](https://github.com/community/community/assets/10833976/1d9f1500-ba40-4e6e-b4e5-a5bab578dfe2)

**Comment by RangHo** [comment]
Note: This only happens on "For You" page only. The "Following" tab works just fine.

**Comment by milesrichardson** [comment]
Seeing the same. When the page first renders, there is no overflow, but then after some number of milliseconds the overflow appears. Here's a screen recording

https://github.com/community/community/assets/835921/a7f02c56-ef66-44a3-8b66-b04dee719995

**Reply by RangHo** [reply]
This is the exact same issue I am seeing. The overflow happens when the For You list is loaded and rendered. That is, after the loading octocat icon disappears and content is rendered.

**Reply by milesrichardson** [reply]
yep. and does not happen on the "following" tab. I expect the delayed rendering bug is because it defaults to the following tab and switches to the for you tab after the scripts load

**Comment by AWqxKAWERbXo** [comment]
I also got this kind of problem.

**Comment by mariofix** [comment]
Same issue using Brave. The search bar is unusable at this point.
